### ISSUE TITLE
Any chance of a how-to in the readme.md?

### ISSUE THREAD

**Issue by iLukey** [erickgirard/Fetlifejs#1]
Hi,

I'm not so great with OAuth so a brief how-to in the readme woud be super helpful. Just a quickstart guide really? 

Gonna have a play now and if I figure it out I'll raise a PR for it :)

Thanks,
Luke

**Comment by witquicked** [comment]
@erickgirard from what I'm seeing here, you designed this to be a node module, acting as a library to facilitate other ES-based projects?

**Comment by erickgirard** [comment]
@witquicked absolutely, the goal is to eventually package it with webpack and put it on npm for both node and web projects. Althought the project is not quite ready for production usage.

**Comment by witquicked** [comment]
Very nice, indeed. I've got a bunch of experience with webpack, and would be glad to help.

Just some thoughts, take them or leave 'em:

I think you could get away with calling it "fetlife" without the "js" suffix. Though that brings up another interesting topic: I don't know if you're affiliated with FL or not, but I'd imagine some kind of Trademark considerations are in play here? In all my years working in FOSS, I've yet to find an ecosystem quite like this one, so I'm unclear the best path forward to ensure longevity and sustainability.
